packages/admintable/src/components/table/components/inputs/InputImage.tsx

```diff
@@ -37,49 +34,43 @@ const InputImage: FC<IInputImage> = ({
       }
     }
   }
-  useEffect(() => {
-  }, [popupRef])
+  useEffect(() => {}, [popupRef])
 
   const handlePopup = () => {
     setSelected(prev => !prev)
-  } 
+  }
 
   return (
-    <ImageWrapper ref={ref as unknown as any}
-    >
-        {isOnScreen && (
-          <>
-            <img
-              src={imageSrc}
-              alt={cellId}
-              width={75}
-              height={75}
-              onClick={handlePopup}
-            /> 
-            {selected && 
-              <AnimateIn>
-                <PopupWrapper ref={popupRef}>
-                  <h3>{userInfo?.name}</h3>
-                  <SrcInput
-                    type="text"  
-                    value={imageSrc}
-                    onChange={e => handleOnChange(e)}
-                  />
-                  <Button
-                    value="Done"
-                    sm
-                    onClick={handlePopup}
-                  />
-                </PopupWrapper>
-              </AnimateIn>}
-          </>
+    <ImageWrapper ref={ref as unknown as any}>
+      <>
+        <img
+          src={imageSrc}
+          alt={cellId}
+          width={75}
+          height={75}
+          onClick={handlePopup}
+          referrerPolicy="no-referrer"
+        />
+        {selected && (
+          <AnimateIn>
+            <PopupWrapper ref={popupRef}>
+              <h3>{userInfo?.name}</h3>
+              <SrcInput
+                type="text"
+                value={imageSrc}
```

Comment:
Same here [reference to message](https://github.com/ACE-VSIT/website/pull/58/files#r927562992)